### DISCUSSION TITLE
Smart Home da Zero: Attuatori o Smart Switches?

### DISCUSSION THREAD

**Discussion by Millyto22** [indomus/forum#2576] [Hardware & Firmware]
Ciao a tutti

Sono nuovo sul Forum quindi spero di star facendo questa domanda nella sezione giusta.
Long story short: sto ristrutturando casa e nel farlo dovrò rifare da zero l'impianto elettrico (la casa era molto vecchia e l'impianto era ancora di quelli con il filo che correva sul muro e non incassato all'interno).
Essendo da molto interessato al mondo della domotica e della smart home, ma non avendone mai messa in piede una avrei bisogno di qualche suggerimento:
Dovendo rifare l'impianto elettrico sto cercando di capire quale possa essere la soluzione migliore per il controllo delle luci.
Mia moglie non è molto smart per cui delle smart lights sono da escludersi; così però il campo si restringe, ma ancora oggi non riesco a decidermi tra l'utilizzo di attuatori (Come gli shelly 2 Plus PM o il Sonoff R3) e l'utilizzo di interruttori nativamente smart (vorrei però evitare le serie proprietarie di BTICINO e VIMAR).
Voi cosa consigliereste?
Contate che in vista della fine dei lavori (erano previsti per fine primavera, forse verranno ultimati a fine inverno sob) sono già in possesso di HomeAssistant Yellow con CM4 da 8GB di RAM, e utilizzerei HA come Hub di gestione di tutti i dispositivi.
Per darvi qualche info in più, nella casa sono previsti 12 punti luce controllati in totale da 21 interruttori (alcuni punti luce saranno controllati da due pulsanti).
Spero di aver presentato il tutto in maniera chiara, aspetto un vostro Feedback per capire come muovermi
Buona Giornata!

Luca

**Comment by Marcunz** [comment]
Io nella tua stessa situazione mi sono detto che 1) pure i commerciali e i tecnici sondati non mi sembrano particolarmente sul pezzo da Vimar a Bticino e in caso di assistenza non so come andrebbe a finire con un sistema chiuso che pago per essere più easy rispetto a HA 2) se tanto alla fine devo passare per HA per avere quello che interessa a me, cioè una cosa tipo Control4 ma che costo zero, tanto vale che scelgo componenti free senza strapagare i vari frutti connessi. Il giorno che si romperà un interruttore o una presa spenderò 5€ per cambiarla e anche la moglie potrà usarlo senza rendersi conto di cosa c’è sotto.

Perciò faccio fare un impianto tradizionale e piano piano lo domotizzo, ovviamente cercando di pensare prima a dove mettere i vari Shelly (se in quadro, se tutti Wi-Fi o se e in quali scatole di derivazione).

poi certo, se hai 40k da spendere fai come Andrea Galeazzi e ti fai un impianto knx con tutti i crismi.
